tools/constants.ts

```diff
@@ -11,3 +11,4 @@ export const MIN_COMMUNITY_TOKENS_TO_CREATE_W_0_SUPPLY = 1000000
  */
 
 export const DISABLED_VOTER_WEIGHT = new BN('18446744073709551615')
+export const DISABLED_VALUE = new BN('18446744073709551615')
```

Comment:
I felt this was more accurate, since this value is used for more than just disabled weights. We should probably not have two names for the same constant though.

Reply:
It's actually used for disabled voter weights only. It denotes the value (`u64::MAX`) which is used to set given option as disabled. You can find more details here https://github.com/solana-labs/solana-program-library/pull/3501

In the codebase you might find legacy names `minTokensToCreateProposal`, `minTokensToCreateGovernance`. They are legacy because the concept of the thresholds and governance power was abstracted away from the number of tokens to abstract weight which might or might not be related to tokens